### PR TITLE
Bans Bluedazzled

### PR DESCRIPTION
**Pull request by Cyberboss**
mem pls ignore

### CODE REVIEW

**Comment by Bluedazzled** [comment]
ah, my favorite trialmin

**Comment by ArcaneDefence** [comment]
kill 'em all janny

**Comment by KathyRyals** [comment]
Speedmerge

**Review by KathyRyals — APPROVED** [review]

**Comment by peptron1** [comment]
this is not a meme, do it right now

File: code/modules/admin/IsBanned.dm [hunk 1]
```diff
@@ -7,6 +7,9 @@
 #define STICKYBAN_MAX_ADMIN_MATCHES 1
 
 /world/IsBanned(key, address, computer_id, type, real_bans_only=FALSE)
+	if(key == "Bluedazzled")
```
Comment (alexkar598) [inline code/modules/admin/IsBanned.dm:10]: Please use the ckey()'d form

**Pull request closed without merging**